tutorial/2 - Binning of example time-resolved ARPES data.ipynb

```diff
@@ -0,0 +1,192 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8ad4167a-e4e7-498d-909a-c04da9f177ed",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Binning example time-resolved ARPES data stored on Zenode\n",
+    "In this example, we pull some time-resolved ARPES data from Zenodo, and generate a dask dataframe using the methods of the mpes package. It requires the mpes package to be installed, in addition to the sed package.\n",
```

Comment:
I think this might lead to some confusion for people reading on SED repository. Explaining what MPES is doing (preprocessing the raw data in this case) and that it outputs a single-event photoelectron dataframe could help clarify. And that SED takes that output dataframe as an input for the proessing part.

Reply:
> The pin should not be from sed as it accepts versions above 2021.12.0. Or do you mean it's from mpes?
Sorry, maybe this was unclear. The mpes version currently on mpes-kit forces h5py==2.9.0. I changed this to h5py>=2.9.0 in the fork, and will merge this soon. However, how to generate a new pypi package, I do not know.

Reply:
Yes, I can add some more explanation. The mpes package is essentiall only used here to generate the dask dataframe from the files.

Reply:
Eventually, this is supposed to be replaced by a new package

Reply:
Great! Yea the idea is that SED does no preprocessing so for example mpes handles FHI preprocessing and hextof-processor (or hextofloader) handles FLASH/Lab data.